htmd/parameterization/fftype.py

```diff
@@ -125,124 +142,114 @@ def fftype(mol, rtfFile=None, prmFile=None, method='GAFF2', acCharges=None, tmpD
                     raise RuntimeError('"antechamber" failed')
 
                 # Run parmchk2
-                returncode = subprocess.call([parmchk2_binary,
-                                              '-f', 'prepi',
-                                              '-s', atomtype,
-                                              '-i', 'mol.prepi',
-                                              '-o', 'mol.frcmod',
-                                              '-a', 'Y'], cwd=tmpdir)
+                cmd = ['parmchk2',
```

Comment:
Why this? I think it's better to check for the binary as it was before.

Reply:
`subprocess.call` it does implicitly

Reply:
It was more to give a more sensible error message to the user but whatever. It's not supposed to be missing anyway